bin/lts-candidate-stats

```diff
@@ -49,13 +49,35 @@ class Script {
 
     static void printListing(String title, items) {
         if (!items.isEmpty()) {
-            println "${title}:"
+            println "${title}\n${''.padRight(title.length(), '-')}\n"
 
             for (it in items) {
-                def prio = it.priority.toString().padRight(10)
-                println "\t${it.key}\t\t${prio}\t\t${ageIndicator(it)}"
-                println "\t\t${it.summary}"
-                println "\t\t${it.link}"
+                def prio = it.priority.toString()
+                if (prio.toLowerCase() in ['blocker', 'critical']) {
+                    prio = Ansi.color(prio.padRight(10), Ansi.RED)
+                } else {
+                    prio = prio.padRight(10)
+                }
+
+                def age = Ansi.color(ageIndicator(it), Ansi.YELLOW)
+                println "${it.key}\t\t${prio}\t\t${age}"
+                def labels = it.labels.label.collect { it.toString() }
+                if ('non-trivial-lts-backporting' in labels){
+                    println "\t${Ansi.color('non-trivial-lts-backporting', Ansi.RED)}"
+                }
+                println "\t${it.summary}"
+                println "\t${it.link}"
+                it.issuelinks.issuelinktype.each {
+                    if (it.name.toString() == 'Cause') {
```

Comment:
Would add a less scary variant (i.e. "else") that shows any other relationships. Adds a bit of noise, but if we only got a "relates to" accidentally, adds safety.

Ideally also obtain linked issue titles. to show here.

Reply:
I gave this a try and it adds more noise than value, IMO. Note the report is not (supposed to be) used _instead_ or reviewing the JIRA but as a condensed report that drives the process. This will obviously evolve to highlight things that are noteworthy, but it by no means mean what does not pop up will be ignored.